core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -974,6 +974,22 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     assert(manager.isZombie)
   }
 
+  test("check uniqueness of TaskSetManager name") {
```

Comment:
Can you make this a little more specific and also include the JIRA: "[SPARK-17894] Verify TaskSetManagers for different stage attempts have unique names"